primefaces/src/main/resources/META-INF/primefaces-p.taglib.xml

```diff
@@ -131,25 +131,25 @@
             org.primefaces.component.export.DataExporterTagHandler
         </handler-class>
         <attribute>
-            <description>Server side id of the datatable whose data would be exported.</description>
+            <description>Search expression to resolve one or multiple datatables.</description>
```

Comment:
DataExporter is not limited to DataTAble, therefore we should say something like "multiple target components"

Reply:
OK, I going to also update docs file so both reflects the same.